mu/ctl/args.py

```diff
@@ -0,0 +1,10 @@
+import argparse
+from mu.ctl.log import pm_logs
+
+def get_args():
+    parser = argparse.ArgumentParser()
```

Comment:
Consider filling in the `description` arguments on the parser/subparser/options for discoverability via `muctl --help`. You can also set `prog='muctl'` so it displays as `Usage: muctl ...` instead of `Usage: control.py ...`.

Also applies to mu/srv/config.py.

Reply:
docs not the biggest priority at the moment since things might change, but yeah added some short lines. I think it probably makes more sense to overhaul help messages at the end though more than do it along the way